pkg/schedule/plan/status.go

```diff
@@ -72,6 +72,7 @@ const (
 	StatusStoreLowSpace = iota + 500
 	// StatusStoreNotExisted represents the store cannot be found in PD.
 	StatusStoreNotExisted
+	StatusStoreSplitRegion
```

Comment:
maybe need to add name as same as `StatusStoreNotExisted`

Reply:
good catch , has fixed.